<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMessageDialog</class>
 <widget class="QDialog" name="NewMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Message</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="fromLabel">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="fromDisclaimer">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="toLabel">
       <property name="text">
        <string>To:</string>
       </property>
       <property name="buddy">
        <cstring>toEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="toEdit">
       <property name="toolTip">
        <string>The message name.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="topicLabel">
       <property name="text">
        <string>Topic:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="topicEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="replyLabel">
       <property name="text">
        <string>Original Message:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPlainTextEdit" name="replyEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_10"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Your Message:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPlainTextEdit" name="messageEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="aliasEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewMessageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewMessageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
